onto_spread_ed/release/GenerateHierarchicalSpreadsheetReleaseStep.py

```
@@ -0,0 +1,180 @@
+            def annotate(n: Node):
+                id = ontology.get_id_for_iri(n.item)
+                fields = {
+                    "comment": "Comment",
+                    "subontology": "Sub-ontology",
+                    "examples": "Examples",
+                    "synonyms": "Synonyms",
+                    "crossreference": "Cross reference",
+                    "informaldefinition": "Informal definition",
+                }
+                for field_key, field in fields.items():
+                    node_data = data.get(id, dict())
```

Ruff

Function definition does not bind loop variable `data`

(B023)